src/ShadowDOM/wrappers/Range.js

```diff
@@ -20,23 +20,69 @@
 
   var OriginalRange = window.Range;
 
+  var ShadowRoot = scope.wrappers.ShadowRoot;
+
+  function isElement(node){
```

Comment:
style note: it looks like spaces are missing between `){` ... I see this in a lot of places.